### PR TITLE
Set up confirmation page

### PR DESCRIPTION
**Pull request by cehsu**
These changes set up the VRE chapter 31 confirmation page and update content for the Chapter 36 confirmation page ([ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8957)).

localhost_3001_employment_vocational-rehab-and-employment_application_chapter31_confirmation

### CODE REVIEW

File: src/js/vre/chapter31/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -8,11 +22,75 @@ class ConfirmationPage extends React.Component {
...
   }
 
   render() {
+    const form = this.props.form;
+    const response = form.submission.response
+      ? form.submission.response.attributes
+      : {};
+    const formData = this.props.form.data || {};
+
+    const name = formData['view:isVeteran']
```
Comment (jbalboni) [inline src/js/vre/chapter31/containers/ConfirmationPage.jsx]: I don't think we have this field in chapter 31?

**Review by jbalboni — APPROVED** [review]
One thing, but LGTM.

**Review by stacyhlinn — APPROVED** [review]

File: src/js/vre/chapter31/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/vre/chapter31/containers/ConfirmationPage.jsx]: Thanks, you're correct. Sorry about the copy pasta, will fix.

**Comment by peggygannon** [comment]
@cehsu @jbalboni Hi, there. Here are my edits to the confirmation page. Let me know if you have any questions about my suggested edits. Thx!

1. header should be sentence case, to match our form style: Apply for vocational rehabilitation

2. Under **Claim received** please add:
We’ve received your application. Thank you for applying for vocational rehabilitation benefits. 

delete: We may contact you....sentence

3.  I'm thinking we might want to move up **What happens after I apply?** to before the blue box (based on our learnings from VIC)
So it would look like this:
**What happens after I apply?**
We process claims in the order we receive them. 
*(please delete: We usually process claims within 1 week.. that is not accurate)*
Add: We may contact you if we have any questions or need more information. 
If you’re eligible for vocational rehabilitation benefits, we’ll invite you to an orientation session at your nearest VA regional office or out base office. 

*Please print this page for your records.*
[[begin blue box]]

4. fix form name and number in the blue box. It should be:
Vocational Rehabilitation claim (Form 28-1900)

5. Under **Need help?** 
please bump down Monday so it's on the same line as Friday

**Review by rianfowler — APPROVED** [review]

**Pull request merged**